src/lib/optimizer/optimizer.cpp

```diff
@@ -124,6 +125,8 @@ std::shared_ptr<Optimizer> Optimizer::create_default_optimizer() {
 
   optimizer->add_rule(std::make_unique<IndexScanRule>());
 
+  optimizer->add_rule(std::make_unique<PredicateMergeRule>(4));
```

Comment:
Why is this an externally exposed parameter, no a parameter within the rule?

Reply:
Because the parameter is set to 1 for testing.

Reply:
Because the parameter is set to 1 for testing.

Reply:
You can also set it in the test without adding it as a parameter that is not used in regular execution:

https://github.com/hyrise/hyrise/blob/f79e2c99deb902bdc8632147ed6183c8cfcd015e/src/test/optimizer/strategy/in_expression_rewrite_rule_test.cpp#L40-L41